src/BlazorUI/Bit.BlazorUI/Styles/Fluent/colors.fluent-light.scss

```
@@ -178,7 +210,23 @@
     --bit-clr-brd-ter-light: #E9E9E9;
     --bit-clr-brd-ter-light-hover: #D7D7D7;
     --bit-clr-brd-ter-light-active: #C1C1C1;
+    --bit-clr-brd-ter-dis: #EEEDEC;
+    --bit-clr-brd-ter-dis-text: #CCCCCC;
     --bit-clr-brd-dis: #E5E3DF;
     //req
```

Stylelint (17.11.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)